src/server.ts

```diff
@@ -51,10 +48,6 @@ export function createServerWithTools(options: Options): Server {
     };
   });
 
-  server.setRequestHandler(ListResourcesRequestSchema, async () => {
```

Comment:
Where did this and ReadResourceRequestSchema go?

Reply:
We don't have resources yet.